.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -15,6 +15,10 @@ steps:
       filePath: yarn-ci.sh
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31205&commitId=b7b1337deafeb4b68239900847bd26b996432438&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|2| Changed|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|7| Changed|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|9| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|1| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|7| Changed|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|7| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|2| Changed|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|2| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|9| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|1| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|2| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|2| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|1| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|1| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|2| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|9| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|1| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|2| Changed|
|Accordion Converged.size.normal.chromium.png|2| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|1| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Avatar Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|13| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|4| Changed|
|Avatar Converged.basic.normal.chromium.png|4| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|5| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|3| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|33| Changed|
|Avatar Converged.color - High Contrast.normal.chromium.png|44| Changed|
|Avatar Converged.color - Dark Mode.normal.chromium.png|7| Changed|
|Avatar Converged.customSize+image.normal.chromium.png|2| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|25| Changed|
|Avatar Converged.color.normal.chromium.png|8| Changed|
|Avatar Converged.color+active.normal.chromium.png|295| Changed|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|403| Changed|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|154| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|27| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|17| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|28| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|14| Changed|

</details>


  <details><summary><b>AvatarGroup Converged</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|1| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|1| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|49| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|2| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|80| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|75| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|82| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|5| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Badge Converged</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.filled - Dark Mode.chromium.png|40| Changed|
|Badge Converged.filled - High Contrast.chromium.png|67| Changed|
|Badge Converged.ghost - High Contrast.chromium.png|51| Changed|
|Badge Converged.outline.chromium.png|24| Changed|
|Badge Converged.filled.chromium.png|46| Changed|
|Badge Converged.outline - High Contrast.chromium.png|50| Changed|
|Badge Converged.size- extra-small.chromium.png|3| Changed|
|Badge Converged.ghost.chromium.png|28| Changed|
|Badge Converged.ghost - Dark Mode.chromium.png|21| Changed|
|Badge Converged.size- medium - RTL.chromium.png|34| Changed|
|Badge Converged.size- extra-large.chromium.png|62| Changed|
|Badge Converged.size- large.chromium.png|69| Changed|
|Badge Converged.size- extra-large - RTL.chromium.png|61| Changed|
|Badge Converged.outline - Dark Mode.chromium.png|17| Changed|
|Badge Converged.size- large - RTL.chromium.png|69| Changed|
|Badge Converged.size- extra-small - RTL.chromium.png|3| Changed|
|Badge Converged.size- small.chromium.png|35| Changed|
|Badge Converged.size- small - RTL.chromium.png|35| Changed|
|Badge Converged.size- medium.chromium.png|34| Changed|
|Badge Converged.tint.chromium.png|25| Changed|
|Badge Converged.tint - Dark Mode.chromium.png|16| Changed|
|Badge Converged.tint - High Contrast.chromium.png|54| Changed|
|Badge Converged.width constrained.chromium.png|5| Changed|

</details>


  <details><summary><b>Breadcrumb Converged</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.appearance - Dark Mode.focused.chromium.png|2| Changed|
|Breadcrumb Converged.appearance - Dark Mode.default.chromium.png|2| Changed|
|Breadcrumb Converged.appearance - Dark Mode.pressed.chromium.png|2| Changed|
|Breadcrumb Converged.appearance - RTL.focused.chromium.png|3| Changed|
|Breadcrumb Converged.appearance.focused.chromium.png|3| Changed|
|Breadcrumb Converged.appearance - RTL.pressed.chromium.png|3| Changed|
|Breadcrumb Converged.appearance - RTL.hover.chromium.png|3| Changed|
|Breadcrumb Converged.appearance - Dark Mode.hover.chromium.png|2| Changed|
|Breadcrumb Converged.appearance - High Contrast.focused.chromium.png|4| Changed|
|Breadcrumb Converged.appearance - High Contrast.default.chromium.png|4| Changed|
|Breadcrumb Converged.appearance.default.chromium.png|3| Changed|
|Breadcrumb Converged.appearance - High Contrast.hover.chromium.png|4| Changed|
|Breadcrumb Converged.appearance.pressed.chromium.png|3| Changed|
|Breadcrumb Converged.appearance - RTL.default.chromium.png|3| Changed|
|Breadcrumb Converged.size.default.chromium.png|8| Changed|
|Breadcrumb Converged.size.focused.chromium.png|8| Changed|
|Breadcrumb Converged.appearance - High Contrast.pressed.chromium.png|4| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|8| Changed|
|Breadcrumb Converged.appearance.hover.chromium.png|3| Changed|
|Breadcrumb Converged.size.hover.chromium.png|8| Changed|

</details>


  <details><summary><b>Button Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Size large.default.chromium.png|4| Changed|
|Button Converged.Size large.pressed.chromium.png|3| Changed|
|Button Converged.Size large.hover.chromium.png|4| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|4| Changed|

</details>


  <details><summary><b>Calendar Compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Default - RTL.default.chromium.png|5| Changed|
|Calendar Compat.Custom icons.default.chromium.png|3| Changed|
|Calendar Compat.Default.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Card Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.size.normal.chromium.png|9| Changed|
|Card Converged.CardHeader.normal.chromium.png|3| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|16| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|12| Changed|
|Card Converged.appearance.normal.chromium.png|12| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|12| Changed|
|Card Converged.orientation.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|3| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 68 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|4| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|4| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|3| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|3| Changed|

</details>


  <details><summary><b>Checkbox Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Converged.large+circular+mixed.default.chromium.png|1| Changed|

</details>


  <details><summary><b>Combobox Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Size- large.default.chromium.png|1| Changed|
|Combobox Converged.Size- large.focused.chromium.png|1| Changed|
|Combobox Converged.Size- large.hover.chromium.png|1| Changed|
|Combobox Converged.With selection.default.chromium.png|2| Changed|
|Combobox Converged.With selection.active option.chromium.png|2| Changed|
|Combobox Converged.With selection.hover.chromium.png|2| Changed|

</details>


  <details><summary><b>CounterBadge Converged - colors</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CounterBadge Converged - colors.default.chromium.png|5| Changed|
|CounterBadge Converged - colors.default - RTL.chromium.png|5| Changed|

</details>


  <details><summary><b>DatePicker Compat</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|5| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|5| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|5| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|5| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|4| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|4| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|9| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|9| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|9| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|4| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|5| Changed|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|5| Changed|
|DatePicker Compat.default.opened.chromium.png|5| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|9| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|4| Changed|
|DatePicker Compat.marked dates.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Dialog</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.title custom action.chromium.png|3| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|3| Changed|
|Dialog.title custom action - RTL.chromium.png|3| Changed|
|Dialog.title custom action - High Contrast.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.default drawer.chromium.png|8| Changed|
|Drawer.inline drawer - High Contrast.chromium.png|13| Changed|
|Drawer.default drawer - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer full - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer full - RTL.chromium.png|8| Changed|
|Drawer.inline drawer - RTL.chromium.png|8| Changed|
|Drawer.default drawer - RTL.chromium.png|8| Changed|
|Drawer.inline drawer - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer full - Dark Mode.chromium.png|7| Changed|
|Drawer.default drawer - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer full.chromium.png|8| Changed|
|Drawer.inline drawer large - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer large - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer large - High Contrast.chromium_1.png|13| Changed|
|Drawer.inline drawer large content scroll - RTL.chromium.png|8| Changed|
|Drawer.inline drawer large - RTL.chromium_1.png|8| Changed|
|Drawer.inline drawer position end - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer large content scroll - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer large - RTL.chromium.png|8| Changed|
|Drawer.inline drawer large - Dark Mode.chromium_1.png|7| Changed|
|Drawer.inline drawer large content scroll - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer large content scroll.chromium.png|8| Changed|
|Drawer.inline drawer position end - RTL.chromium.png|8| Changed|
|Drawer.inline drawer position end.chromium.png|8| Changed|
|Drawer.inline drawer position end - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer large.chromium.png|8| Changed|
|Drawer.inline drawer position start - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer position start - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer large.chromium_1.png|8| Changed|
|Drawer.inline drawer position start - RTL.chromium.png|8| Changed|
|Drawer.inline drawer separator position end - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer position start.chromium.png|8| Changed|
|Drawer.inline drawer separator position end - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer separator position end - RTL.chromium.png|8| Changed|
|Drawer.inline drawer separator position start - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer separator position start - High Contrast.chromium.png|13| Changed|
|Drawer.inline drawer separator position end.chromium.png|8| Changed|
|Drawer.inline drawer separator position start - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer - Dark Mode.chromium.png|7| Changed|
|Drawer.inline drawer separator position start.chromium.png|8| Changed|
|Drawer.overlay drawer - RTL.chromium.png|8| Changed|
|Drawer.inline drawer.chromium.png|8| Changed|
|Drawer.overlay drawer alert - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer alert - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer alert.chromium.png|8| Changed|
|Drawer.overlay drawer full - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer alert - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer full - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer large - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer large - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer large content scroll - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer full - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer large - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer large.chromium.png|8| Changed|
|Drawer.overlay drawer full.chromium.png|8| Changed|
|Drawer.overlay drawer large content scroll - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer large content scroll - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer large content scroll.chromium.png|8| Changed|
|Drawer.overlay drawer medium - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer medium.chromium.png|8| Changed|
|Drawer.overlay drawer medium - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer position end - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer non modal - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer position end - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer non modal.chromium.png|8| Changed|
|Drawer.overlay drawer medium - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer position start - High Contrast.chromium.png|13| Changed|
|Drawer.overlay drawer non modal - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer position start - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer non modal - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer position end.chromium.png|8| Changed|
|Drawer.overlay drawer position start - Dark Mode.chromium.png|7| Changed|
|Drawer.overlay drawer position start.chromium.png|8| Changed|
|Drawer.overlay drawer position end - RTL.chromium.png|8| Changed|
|Drawer.overlay drawer.chromium.png|8| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Size- large.default.chromium.png|1| Changed|

</details>


  <details><summary><b>Field</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Combobox+horizontal.default.chromium.png|2| Changed|
|Field.Dropdown+horizontal.default.chromium.png|2| Changed|
|Field.Combobox.default.chromium.png|2| Changed|
|Field.ProgressBar.default.chromium.png|2| Changed|
|Field.Dropdown.default.chromium.png|2| Changed|
|Field.Select+horizontal.default.chromium.png|2| Changed|
|Field.Slider.default.chromium.png|1| Changed|
|Field.Switch.default.chromium.png|3| Changed|
|Field.Select.default.chromium.png|2| Changed|
|Field.SpinButton.default.chromium.png|3| Changed|
|Field.validationMessageIcon.default.chromium.png|1| Changed|

</details>


  <details><summary><b>InfoLabel</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.default - High Contrast.rest.chromium.png|2| Changed|
|InfoLabel.default - RTL.rest.chromium.png|1| Changed|
|InfoLabel.required - RTL.rest.chromium.png|1| Changed|
|InfoLabel.size-large.rest.chromium.png|2| Changed|
|InfoLabel.default.rest.chromium.png|1| Changed|
|InfoLabel.required.rest.chromium.png|1| Changed|
|InfoLabel.wrap.rest.chromium.png|1| Changed|
|InfoLabel.sizes -- open.rest.chromium.png|83| Changed|

</details>


  <details><summary><b>InteractionTag Converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InteractionTag Converged.Brand - High Contrast.pressed content.chromium.png|1| Changed|
|InteractionTag Converged.Brand - High Contrast.default.chromium.png|1| Changed|
|InteractionTag Converged.Brand - High Contrast.hover content.chromium.png|1| Changed|
|InteractionTag Converged.Circular Dismissible.focus dismiss.chromium.png|4| Changed|
|InteractionTag Converged.Circular Dismissible - RTL.focus dismiss.chromium.png|3| Changed|
|InteractionTag Converged.Filled - High Contrast.default.chromium.png|1| Changed|
|InteractionTag Converged.Circular With Secondary Text.focus content.chromium.png|7| Changed|
|InteractionTag Converged.Filled - High Contrast.pressed content.chromium.png|1| Changed|
|InteractionTag Converged.Filled - High Contrast.hover content.chromium.png|1| Changed|
|InteractionTag Converged.Outline - High Contrast.default.chromium.png|1| Changed|
|InteractionTag Converged.Outline - High Contrast.hover content.chromium.png|1| Changed|
|InteractionTag Converged.Outline - High Contrast.pressed content.chromium.png|1| Changed|
|InteractionTag Converged.Size Extra Small With Icon.default.chromium.png|2| Changed|
|InteractionTag Converged.Size Small With Icon.default.chromium.png|2| Changed|

</details>


  <details><summary><b>Menu Converged - MenuItemLinks</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - MenuItemLinks.default.hover menuitemlink.chromium.png|4| Changed|
|Menu Converged - MenuItemLinks.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Menu Converged - basic</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default - RTL.chromium.png|5| Changed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|4| Changed|
|Menu Converged - basic.default.hover menuitem.chromium.png|4| Changed|
|Menu Converged - basic.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.default - RTL.chromium.png|10| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|9| Changed|
|Menu Converged - groups.default.chromium.png|10| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|16| Changed|

</details>


  <details><summary><b>Menu Converged - long content</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - long content.default - RTL.chromium.png|2| Changed|
|Menu Converged - long content.default.chromium.png|2| Changed|

</details>


  <details><summary><b>Menu Converged - secondary content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - secondary content.default - RTL.chromium.png|5| Changed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|4| Changed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|4| Changed|
|Menu Converged - secondary content.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|7| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|5| Changed|
|Menu Converged - selection.checkbox.chromium.png|5| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|4| Changed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|5| Changed|
|Menu Converged - selection.switch - Dark Mode.chromium.png|3| Changed|
|Menu Converged - selection.checkbox.selected.chromium.png|5| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|8| Changed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|4| Changed|
|Menu Converged - selection.switch.chromium.png|5| Changed|
|Menu Converged - selection.switch.selected.chromium.png|5| Changed|
|Menu Converged - selection.switch - RTL.selected.chromium.png|5| Changed|
|Menu Converged - selection.switch - Dark Mode.selected.chromium.png|2| Changed|
|Menu Converged - selection.switch - High Contrast.chromium.png|7| Changed|
|Menu Converged - selection.switch - RTL.chromium.png|5| Changed|
|Menu Converged - selection.switch - High Contrast.selected.chromium.png|4| Changed|

</details>


  <details><summary><b>Menu Converged - selection groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection groups.default - RTL.chromium.png|10| Changed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|10| Changed|
|Menu Converged - selection groups.default.chromium.png|10| Changed|
|Menu Converged - selection groups.default.selected.chromium.png|10| Changed|

</details>


  <details><summary><b>MenuButton Converged</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|5| Changed|
|MenuButton Converged.Size large.hover.chromium.png|4| Changed|
|MenuButton Converged.Size small.hover.chromium.png|1| Changed|
|MenuButton Converged.Size small.default.chromium.png|1| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|1| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|1| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|3| Changed|
|MenuButton Converged.Size large.default.chromium.png|5| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|1| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|1| Changed|
|MenuButton Converged.With icon.default.chromium.png|1| Changed|
|MenuButton Converged.With icon.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>MessageBar</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Auto.default.chromium.png|1| Changed|
|MessageBar.Intents - High Contrast.default.chromium.png|10| Changed|
|MessageBar.Intents - RTL.default.chromium.png|5| Changed|
|MessageBar.Intents.default.chromium.png|5| Changed|
|MessageBar.Intents - Dark Mode.default.chromium.png|3| Changed|
|MessageBar.Multiline.default.chromium.png|5| Changed|
|MessageBar.Multiline No Actions.default.chromium.png|4| Changed|
|MessageBar.Square.default.chromium.png|1| Changed|

</details>


  <details><summary><b>Persona Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.basic - High Contrast.normal.chromium.png|4| Changed|
|Persona Converged.textPosition.normal.chromium.png|2| Changed|
|Persona Converged.textAlignment.normal.chromium.png|1| Changed|
|Persona Converged.size-presence.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>Popover Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|24| Changed|

</details>


  <details><summary><b>Positioning</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|32| Changed|
|Positioning.Multiple scroll parents.container attached to target.chromium.png|1| Changed|
|Positioning.Multiple scroll parents.chromium.png|1| Changed|
|Positioning.auto size with async content reset styles on updatePosition.chromium.png|2| Changed|
|Positioning.arrow.chromium.png|28| Changed|
|Positioning.fallback positioning.chromium.png|1| Changed|
|Positioning.auto size with async content reset styles on updatePosition.floating element width fills boundary and overflows 10px because of overflow-clip.chromium.png|2| Changed|
|Positioning.disable tether.chromium.png|2| Changed|
|Positioning.visibility modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|2| Changed|
|Positioning.visibility modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|2| Changed|
|Positioning.visibility modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|2| Changed|

</details>


  <details><summary><b>Positioning (no decorator)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning (no decorator).scroll jumps.chromium.png|66| Changed|
|Positioning (no decorator).scroll jumps.positions without scroll jump.chromium.png|111| Changed|
|Positioning (no decorator).scroll jumps (with context usage).chromium.png|73| Changed|
|Positioning (no decorator).scroll jumps (with context usage).positions without scroll jump.chromium.png|101| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - OOF status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|7| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|8| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - inverted background</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - inverted background.default - High Contrast.default.chromium.png|233| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.default.chromium.png|48| Changed|
|PresenceBadge Converged - inverted background.default.default.chromium.png|51| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - sizes</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|5| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|4| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - status</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - status.default - RTL.chromium.png|7| Changed|
|PresenceBadge Converged - status.default.chromium.png|7| Changed|

</details>


  <details><summary><b>Rating Converged</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating Converged.Brand Rating with half star - Dark Mode.hover.chromium.png|1| Changed|
|Rating Converged.Brand Rating with half star - Dark Mode.rest.chromium.png|1| Changed|
|Rating Converged.Brand Rating with half star - High Contrast.rest.chromium.png|13| Changed|
|Rating Converged.Brand Rating with half star - High Contrast.hover.chromium.png|19| Changed|
|Rating Converged.Brand Rating with half star.rest.chromium.png|6| Changed|
|Rating Converged.Marigold Rating with half star - High Contrast.hover.chromium.png|11| Changed|
|Rating Converged.Marigold Rating with half star - Dark Mode.hover.chromium.png|8| Changed|
|Rating Converged.Brand Rating with half star.hover.chromium.png|9| Changed|
|Rating Converged.Neutral Rating with half star - Dark Mode.hover.chromium.png|14| Changed|
|Rating Converged.Neutral Rating with half star - High Contrast.hover.chromium.png|19| Changed|
|Rating Converged.Marigold Rating with half star - Dark Mode.rest.chromium.png|6| Changed|
|Rating Converged.Marigold Rating with half star.rest.chromium.png|1| Changed|
|Rating Converged.Neutral Rating with half star - Dark Mode.rest.chromium.png|10| Changed|
|Rating Converged.Marigold Rating with half star - High Contrast.rest.chromium.png|8| Changed|
|Rating Converged.Neutral Rating with half star - High Contrast.rest.chromium.png|13| Changed|
|Rating Converged.Rating size large.rest.chromium.png|10| Changed|
|Rating Converged.Marigold Rating with half star.hover.chromium.png|2| Changed|
|Rating Converged.Rating size extra-large.rest.chromium.png|17| Changed|
|Rating Converged.Rating size medium.rest.chromium.png|6| Changed|
|Rating Converged.Neutral Rating with half star.rest.chromium.png|11| Changed|
|Rating Converged.Neutral Rating with half star.hover.chromium.png|15| Changed|
|Rating Converged.Rating with custom square icons.rest.chromium.png|8| Changed|
|Rating Converged.Rating size small.rest.chromium.png|8| Changed|
|Rating Converged.Rating with custom circle icons.rest.chromium.png|18| Changed|

</details>


  <details><summary><b>RatingDisplay Converged</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|1| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|1| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|2| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|3| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|1| Changed|
|RatingDisplay Converged.custom circle icons.rest.chromium.png|5| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|3| Changed|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|2| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|8| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|3| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|4| Changed|
|RatingDisplay Converged.custom square icons.rest.chromium.png|5| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|4| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|3| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|2| Changed|

</details>


  <details><summary><b>SearchBox Converged</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox Converged.Clears value.input focused.chromium.png|74| Changed|
|SearchBox Converged.Size- large.default.chromium.png|1| Changed|
|SearchBox Converged.Size- small.focused.chromium.png|1| Changed|
|SearchBox Converged.Size- small.default.chromium.png|1| Changed|
|SearchBox Converged.Size- small.hover.chromium.png|1| Changed|
|SearchBox Converged.Size- large.focused.chromium.png|3| Changed|
|SearchBox Converged.Size- large.hover.chromium.png|3| Changed|
|SearchBox Converged.With custom dismiss - RTL.hover.chromium.png|2| Changed|
|SearchBox Converged.With custom dismiss - RTL.focused.chromium.png|2| Changed|
|SearchBox Converged.With custom dismiss.focused.chromium.png|1| Changed|
|SearchBox Converged.With custom dismiss.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>Select Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Select Converged.Size- large.default.chromium.png|1| Changed|

</details>


  <details><summary><b>Skeleton converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|2| Changed|
|Skeleton converged.Translucent Skeleton with circle - High Contrast.chromium.png|2| Changed|

</details>


  <details><summary><b>Slider Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider Converged.Root - High Contrast.pressed.chromium.png|1| Changed|

</details>


  <details><summary><b>SpinButton Converged</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|1| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|1| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|1| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|1| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|1| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|1| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|1| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|1| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|1| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|1| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|1| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|1| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|1| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|1| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|1| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|1| Changed|

</details>


  <details><summary><b>Spinner converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Animation - RTL.chromium.png|4| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|64| Changed|
|SwatchPicker Converged.Spacing.default.chromium.png|30| Changed|
|SwatchPicker Converged.spacing.default.chromium.png|52| Changed|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|6| Changed|
|SwatchPicker Converged.default.default.chromium.png|15| Changed|
|SwatchPicker Converged.default - RTL.default.chromium.png|15| Changed|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|6| Changed|

</details>


  <details><summary><b>Switch Converged</b> 50 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch Converged.Disabled and checked - High Contrast.hover.chromium.png|1| Changed|
|Switch Converged.Disabled and checked - High Contrast.pressed.chromium.png|1| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.hover.chromium.png|1| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.pressed.chromium.png|1| Changed|
|Switch Converged.Enabled and checked - Dark Mode.hover.chromium.png|2| Changed|
|Switch Converged.Enabled and checked - Dark Mode.default.chromium.png|1| Changed|
|Switch Converged.Enabled and checked - High Contrast.hover.chromium.png|2| Changed|
|Switch Converged.Enabled and checked - High Contrast.pressed.chromium.png|2| Changed|
|Switch Converged.Enabled and checked - RTL.hover.chromium.png|3| Changed|
|Switch Converged.Enabled and checked - RTL.default.chromium.png|3| Changed|
|Switch Converged.Enabled and checked - High Contrast.default.chromium.png|2| Changed|
|Switch Converged.Enabled and checked - RTL.pressed.chromium.png|3| Changed|
|Switch Converged.Enabled and checked.pressed.chromium.png|3| Changed|
|Switch Converged.Enabled and checked.hover.chromium.png|3| Changed|
|Switch Converged.Enabled and checked.default.chromium.png|3| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.pressed.chromium.png|1| Changed|
|Switch Converged.Disabled and unchecked - High Contrast.default.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.hover.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked - RTL.default.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked - RTL.hover.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked.default.chromium.png|1| Changed|
|Switch Converged.Disabled and checked - High Contrast.default.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked - High Contrast.default.chromium.png|2| Changed|
|Switch Converged.Enabled and unchecked.hover.chromium.png|1| Changed|
|Switch Converged.Required.pressed.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked.pressed.chromium.png|1| Changed|
|Switch Converged.Required.default.chromium.png|1| Changed|
|Switch Converged.With label above - RTL.pressed.chromium.png|1| Changed|
|Switch Converged.With label above - RTL.hover.chromium.png|1| Changed|
|Switch Converged.With label above - RTL.default.chromium.png|1| Changed|
|Switch Converged.Enabled and unchecked - RTL.pressed.chromium.png|1| Changed|
|Switch Converged.With label above.default.chromium.png|1| Changed|
|Switch Converged.With label above.pressed.chromium.png|1| Changed|
|Switch Converged.With label above.hover.chromium.png|1| Changed|
|Switch Converged.With label after - RTL.default.chromium.png|1| Changed|
|Switch Converged.With label after - RTL.pressed.chromium.png|1| Changed|
|Switch Converged.Required.hover.chromium.png|1| Changed|
|Switch Converged.With label after.hover.chromium.png|1| Changed|
|Switch Converged.With label after.default.chromium.png|1| Changed|
|Switch Converged.With label after.pressed.chromium.png|1| Changed|
|Switch Converged.With label before - RTL.default.chromium.png|1| Changed|
|Switch Converged.With label before - RTL.pressed.chromium.png|1| Changed|
|Switch Converged.With label before.pressed.chromium.png|1| Changed|
|Switch Converged.With label before - RTL.hover.chromium.png|1| Changed|
|Switch Converged.With label after - RTL.hover.chromium.png|1| Changed|
|Switch Converged.With label before.hover.chromium.png|1| Changed|
|Switch Converged.With label before.default.chromium.png|1| Changed|
|Switch Converged.With label wrapping.default.chromium.png|1| Changed|
|Switch Converged.With label wrapping.hover.chromium.png|1| Changed|
|Switch Converged.With label wrapping.pressed.chromium.png|1| Changed|

</details>


  <details><summary><b>Table flex - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table flex - subtle selection.rest.chromium.png|14| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|13| Changed|

</details>


  <details><summary><b>Table layout flex</b> 43 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|14| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|26| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|13| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|13| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|13| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|13| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|25| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|13| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|12| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|18| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|13| Changed|
|Table layout flex.multiselect - RTL.chromium.png|13| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|24| Changed|
|Table layout flex.multiselect.chromium.png|13| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|12| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|14| Changed|
|Table layout flex.primary cell.chromium.png|24| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|13| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|13| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|22| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|13| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|13| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|12| Changed|
|Table layout flex.single select - High Contrast.chromium.png|24| Changed|
|Table layout flex.single select (checked).chromium.png|14| Changed|
|Table layout flex.size - extra small.chromium.png|22| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|24| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|13| Changed|
|Table layout flex.single select.chromium.png|13| Changed|
|Table layout flex.single select - RTL.chromium.png|13| Changed|
|Table layout flex.size - medium.chromium.png|13| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|12| Changed|
|Table layout flex.size - small.chromium.png|13| Changed|
|Table layout flex.size - medium - RTL.chromium.png|13| Changed|

</details>


  <details><summary><b>Table layout flex - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|24| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|38| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|28| Changed|
|Table layout flex - cell actions.always visible.chromium.png|28| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|24| Changed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|12| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|45| Changed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|16| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|22| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|14| Changed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|24| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|13| Changed|
|Table layout flex - cell actions.default.chromium.png|13| Changed|
|Table layout flex - cell actions.default.hover row.chromium.png|16| Changed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|16| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|13| Changed|

</details>


  <details><summary><b>Table layout flex - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|17| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|30| Changed|
|Table layout flex - headers.sortable.hover header.chromium.png|19| Changed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|17| Changed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|16| Changed|
|Table layout flex - headers.sortable.chromium.png|19| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|32| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|30| Changed|
|Table layout flex - headers.sortable.press header.chromium.png|19| Changed|

</details>


  <details><summary><b>Table layout flex - resizable columns</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - resizable columns.default.chromium.png|5| Changed|
|Table layout flex - resizable columns.end.chromium.png|10| Changed|

</details>


  <details><summary><b>Table layout flex - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.default (disabled).chromium.png|13| Changed|
|Table layout flex - truncate.false.chromium.png|13| Changed|
|Table layout flex - truncate.true.chromium.png|13| Changed|

</details>


  <details><summary><b>Table layout table</b> 43 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|14| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|13| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|26| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|13| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|13| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|25| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|18| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|13| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|12| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|12| Changed|
|Table layout table.multiselect - RTL.chromium.png|13| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|24| Changed|
|Table layout table.multiselect.chromium.png|13| Changed|
|Table layout table.primary cell.chromium.png|24| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|13| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|13| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|14| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|13| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - brand.chromium.png|13| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|12| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|22| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|13| Changed|
|Table layout table.single select - High Contrast.chromium.png|24| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|13| Changed|
|Table layout table.single select - Dark Mode.chromium.png|12| Changed|
|Table layout table.single select.chromium.png|13| Changed|
|Table layout table.single select (checked).chromium.png|14| Changed|
|Table layout table.single select - RTL.chromium.png|13| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|12| Changed|
|Table layout table.size - small.chromium.png|13| Changed|
|Table layout table.size - extra small.chromium.png|22| Changed|
|Table layout table.size - medium.chromium.png|13| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|24| Changed|
|Table layout table.size - medium - RTL.chromium.png|13| Changed|

</details>


  <details><summary><b>Table layout table - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.always visible - RTL.chromium.png|28| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|38| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|24| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|24| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|45| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.always visible.chromium.png|28| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|13| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|12| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|24| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|14| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|22| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|16| Changed|
|Table layout table - cell actions.in header cell.chromium.png|13| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|16| Changed|
|Table layout table - cell actions.default.chromium.png|13| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|16| Changed|

</details>


  <details><summary><b>Table layout table - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - headers.sortable - Dark Mode.chromium.png|17| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|16| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|17| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|30| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|32| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|30| Changed|
|Table layout table - headers.sortable.chromium.png|19| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|19| Changed|
|Table layout table - headers.sortable.press header.chromium.png|19| Changed|

</details>


  <details><summary><b>Table layout table - resizable columns</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - resizable columns.end.chromium.png|10| Changed|
|Table layout table - resizable columns.default.chromium.png|5| Changed|

</details>


  <details><summary><b>Table layout table - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - truncate.default (disabled).chromium.png|13| Changed|
|Table layout table - truncate.false.chromium.png|13| Changed|
|Table layout table - truncate.true.chromium.png|13| Changed|

</details>


  <details><summary><b>Table table - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table table - subtle selection.rest.chromium.png|14| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|13| Changed|

</details>


  <details><summary><b>Tag Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tag Converged.Brand - High Contrast.default.chromium.png|1| Changed|
|Tag Converged.Filled - High Contrast.default.chromium.png|1| Changed|
|Tag Converged.Outline - High Contrast.default.chromium.png|1| Changed|
|Tag Converged.Size Extra Small With Icon.default.chromium.png|2| Changed|
|Tag Converged.Size Small With Icon.default.chromium.png|2| Changed|

</details>


  <details><summary><b>TagPicker</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.appearance - High Contrast.chromium.png|2| Changed|
|TagPicker.appearance.chromium.png|1| Changed|
|TagPicker.appearance - RTL.chromium.png|1| Changed|
|TagPicker.expand icon - High Contrast.chromium.png|5| Changed|
|TagPicker.disabled - High Contrast.chromium.png|7| Changed|
|TagPicker.disabled - Dark Mode.chromium.png|4| Changed|
|TagPicker.expand icon - RTL.chromium.png|3| Changed|
|TagPicker.expand icon.chromium.png|3| Changed|
|TagPicker.disabled.chromium.png|3| Changed|
|TagPicker.disabled - RTL.chromium.png|3| Changed|
|TagPicker.expand icon - Dark Mode.chromium.png|2| Changed|
|TagPicker.secondary action - RTL.chromium.png|1| Changed|
|TagPicker.size - Dark Mode.chromium.png|1| Changed|
|TagPicker.secondary action - Dark Mode.chromium.png|1| Changed|
|TagPicker.size - High Contrast.chromium.png|3| Changed|
|TagPicker.secondary action - High Contrast.chromium.png|2| Changed|
|TagPicker.size - RTL.chromium.png|2| Changed|
|TagPicker.size.chromium.png|2| Changed|
|TagPicker.secondary action.chromium.png|1| Changed|

</details>


  <details><summary><b>Toast</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast - Dark Mode.Toast visible.chromium.png|3| Changed|
|Toast.Full Toast - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Full Toast.Toast visible.chromium.png|6| Changed|
|Toast.Full Toast - RTL.Toast visible.chromium.png|6| Changed|
|Toast.Full Toast Inverted - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Full Toast Inverted.Toast visible.chromium.png|4| Changed|
|Toast.Full Toast Inverted - Dark Mode.Toast visible.chromium.png|7| Changed|
|Toast.Title Only - Dark Mode.Toast visible.chromium.png|3| Changed|
|Toast.Title Only Inverted - Dark Mode.Toast visible.chromium.png|7| Changed|
|Toast.Title Only - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Title Only - RTL.Toast visible.chromium.png|6| Changed|
|Toast.Title Only Inverted.Toast visible.chromium.png|4| Changed|
|Toast.Title Only.Toast visible.chromium.png|6| Changed|
|Toast.Toast Title Word Break.Toast visible.chromium.png|2| Changed|
|Toast.Title Only Inverted - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Without Subtitle - Dark Mode.Toast visible.chromium.png|3| Changed|
|Toast.Without Subtitle - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Without Subtitle Inverted - Dark Mode.Toast visible.chromium.png|7| Changed|
|Toast.Without Subtitle - RTL.Toast visible.chromium.png|6| Changed|
|Toast.Without Subtitle.Toast visible.chromium.png|6| Changed|
|Toast.Without Subtitle Inverted - High Contrast.Toast visible.chromium.png|10| Changed|
|Toast.Without Subtitle Inverted.Toast visible.chromium.png|4| Changed|

</details>


  <details><summary><b>ToggleButton Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|4| Changed|
|ToggleButton Converged.Size large.default.chromium.png|4| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|4| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|3| Changed|

</details>


  <details><summary><b>Toolbar Converged</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Default.Button Pressed.chromium.png|4| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|4| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|4| Changed|
|Toolbar Converged.Large.default.chromium.png|5| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|4| Changed|
|Toolbar Converged.Default.default.chromium.png|5| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|4| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|4| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|3| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|2| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|2| Changed|
|Toolbar Converged.Small.default.chromium.png|5| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|4| Changed|
|Toolbar Converged.Vertical.default.chromium.png|5| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|4| Changed|
|Toolbar Converged.Far Group.chromium.png|6| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|13| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|1| Changed|

</details>


  <details><summary><b>Tree</b> 33 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.actions - High Contrast.chromium.png|11| Changed|
|Tree.actions - Dark Mode.chromium.png|6| Changed|
|Tree.actions.chromium.png|7| Changed|
|Tree.appearance - Dark Mode.Subtle alpha tree mousedown.chromium.png|257| Changed|
|Tree.appearance - Dark Mode.Transparent tree hover.chromium.png|257| Changed|
|Tree.appearance - Dark Mode.Subtle alpha tree hover.chromium.png|257| Changed|
|Tree.actions - RTL.chromium.png|7| Changed|
|Tree.appearance - High Contrast.Subtle alpha tree hover.chromium.png|256| Changed|
|Tree.appearance - High Contrast.Subtle alpha tree mousedown.chromium.png|256| Changed|
|Tree.appearance - High Contrast.Transparent tree hover.chromium.png|257| Changed|
|Tree.appearance - High Contrast.Transparent tree mousedown.chromium.png|257| Changed|
|Tree.appearance - Dark Mode.Transparent tree mousedown.chromium.png|257| Changed|
|Tree.appearance - RTL.Subtle alpha tree hover.chromium.png|256| Changed|
|Tree.appearance - RTL.Subtle alpha tree mousedown.chromium.png|256| Changed|
|Tree.appearance - RTL.Transparent tree hover.chromium.png|256| Changed|
|Tree.appearance.Subtle alpha tree hover.chromium.png|256| Changed|
|Tree.appearance - RTL.Transparent tree mousedown.chromium.png|256| Changed|
|Tree.appearance.Transparent tree hover.chromium.png|256| Changed|
|Tree.appearance.Subtle alpha tree mousedown.chromium.png|256| Changed|
|Tree.appearance.Transparent tree mousedown.chromium.png|256| Changed|
|Tree.expand icon.chromium.png|1| Changed|
|Tree.expand icon - High Contrast.chromium.png|3| Changed|
|Tree.expand icon - RTL.chromium.png|1| Changed|
|Tree.flat tree single selection - High Contrast.chromium.png|1| Changed|
|Tree.flat tree multi selection - High Contrast.chromium.png|1| Changed|
|Tree.layout - RTL.chromium.png|7| Changed|
|Tree.layout - Dark Mode.chromium.png|4| Changed|
|Tree.layout.chromium.png|7| Changed|
|Tree.layout - High Contrast.chromium.png|9| Changed|
|Tree.persona - Dark Mode.chromium.png|1| Changed|
|Tree.persona - RTL.chromium.png|5| Changed|
|Tree.persona - High Contrast.chromium.png|14| Changed|
|Tree.persona.chromium.png|5| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>